coala_quickstart/coala_quickstart.py

```diff
@@ -24,7 +24,11 @@
 from coala_quickstart.generation.Settings import (
     generate_settings, write_coafile)
 from coala_quickstart.generation.SettingsClass import (
-    collect_bear_settings)
+    collect_bear_settings, build_bear_settings)
```

Comment:
can you make the commit message more descriptive, the commit has 1070 lines of changes so a description would be helpful in reviewing efficiently.

Reply:
yeah I will try my best but it would still be a pain for you, I will make sure I keep committing with smaller sets of changes from next time onwards 😅

Reply:
amended commit message